rocky/tests/objects/test_objects_add.py

```diff
@@ -16,9 +16,9 @@ def test_add_ooi(rf, client_member, mock_organization_view_octopoes, mock_bytes_
     assert response.url == "/en/test/objects/detail/?ooi_id=Network%7Ctestnetwork"
 
     data_without_valid_time = (
-        b'[{"ooi":{"object_type":"Network","scan_profile":null,"user_id":225,'
-        b'"primary_key":"Network|testnetwork","name":"testnetwork"}'
-    )
+        f'[{{"ooi":{{"object_type":"Network","scan_profile":null,"user_id":{client_member.user.id},'
+        f'"primary_key":"Network|testnetwork","name":"testnetwork"}}}}]'
```

Comment:
It does a partial match, so it shouldn't have the last `}]`. It might be good to add a comment to explain that the match is supposed to be partial, but I can't modify unchanged lines in a suggestion.

I'm fine with accepting this, but the other option was to use bytes printf style formatting:
https://docs.python.org/3/library/stdtypes.html#printf-style-bytes-formatting 

```suggestion
        f'"primary_key":"Network|testnetwork","name":"testnetwork"}}'
```

Reply:
Good catch!

Reply:
Nice work guys :) Thank you!